index.js

```diff
@@ -0,0 +1,32 @@
+/*canvas-background*/
+
+document.addEventListener("DOMContentLoaded", function() {
+    const canvas = document.getElementById("myCanvas");
+    const ctx = canvas.getContext("2d");
+
+/*functions*/
+
+    function drawBall(ctx, centerX, centerY, radius, fillColor) {
```

Comment:
this function does not need to live inside callback of "DOMContentLoaded", it better sit on the base

Reply:
I can't seem to make it work if it's not inside that callback, even though my canvas is inside the body in html. I get an error if I don't put it inside "DOMContentLoaded"